### ISSUE TITLE
where is it ?

### ISSUE THREAD

**Issue by rofl0r** [lutris/libfswindow#1]
i just see a LICENSE file, but no lib, and no instructions...

**Comment by rofl0r** [comment]
i'm actually quite interested into this topic, since on the laptop i use atm i can't get any game to stretch to fullscreen properly, instead the call to SDL_WM_Toggle_Fullscreen() results only in a black border around the tiny game window (for example 320x240).

**Comment by strycore** [comment]
yeah, that project didn't go anywhere but you're lucky because another one did!

Try SDLCL: https://github.com/MrAlert/sdlcl it's not compatible with *every* game but I've managed to get it working with Prey, UT, Rune and others have used it with Dungeons of Dredmor, Psychonauts, Quake 4, … The only game I still can't play with this is Jamestown.

**Comment by rofl0r** [comment]
and using SDL2 instead of SDL 1.2 should fix exactly that issue, of not getting stretch-to-fullscreen effect ?

**Comment by rofl0r** [comment]
btw, when you talk about UT, do you mean Unreal Tournament 99? which afaik is only available in a closed-source linux port by loki games ?

**Comment by strycore** [comment]
yes, this lib acts as a drop in replacement for SDL 1.2, using SDL 2 stuff. 

I've indeed run UT 99 (linux binary available on Lutris) and I have yet to try it out with UT 2003/2004.
